### ISSUE TITLE
Request slack notification card message

### ISSUE THREAD

**Issue by dyson-yamashita** [fossamagna/amplify-category-console-notification#12]
**Is your feature request related to a problem? Please describe.**
Notification card message is "This content can't be displayed”.
I want to know build status by notification card message.

**Describe the solution you'd like**
Display message. ex: AWS Amplify Console Build STARTED (develop)

**Describe alternatives you've considered**
Add text attribute in slack message.
```
return {
      text: `AWS Amplify Console Build ${jobStatus} (${branchName})`, #ADD text attribute
      blocks: [
        {
          type: "section",
          text: {
            type: "mrkdwn",
            text: `AWS Amplify Console Build <${jobUrl}|#${jobId}>\nStatus: *${jobStatus}* ${emoji} \n<${appUrl}|Open your App (${branchName})>`
          }
        }
      ]
    }
```

**Additional context**

**Comment by dyson-yamashita** [comment]
Create a pull request.
Please consider the function.

**Comment by fossamagna** [comment]
@dyson-yamashita I released v0.3.`. Please try it.

**Comment by dyson-yamashita** [comment]
@fossamagna Thank you so much.
I was able to confirm the reflection.
